tenacity/tests/test_tenacity.py

```diff
@@ -172,9 +172,8 @@ def _raise_try_again(self):
 
     def test_retry_try_again(self):
         self._attempts = 0
-        r = Retrying(stop=tenacity.stop_after_attempt(5),
-                     retry=tenacity.retry_never).call(
-            self._raise_try_again)
+        Retrying(stop=tenacity.stop_after_attempt(5),
```

Comment:
I was running flake8 on my change, and this was 'an issue to flake8' so I just fixed it :-P